components/brave_wallet_ui/components/desktop/popup-modals/edit-visible-assets-modal/index.tsx

```diff
@@ -172,7 +174,8 @@ const EditVisibleAssetsModal = (props: Props) => {
     symbol: selectedNetwork.symbol,
     visible: true,
     tokenId: '',
-    coingeckoId: ''
+    coingeckoId: '',
```

Comment:
I believe we have a helper for this? Also we could add `chainId` from the network here.